README.md

```diff
@@ -150,6 +159,33 @@ simp_bolt::config::modulepath: /path/to/modules
 To apply an existing manifest, `su` to the bolt user and execute
 `bolt apply <manifest> --nodes <NODE NAME(S)> --password --sudo-password`.
 
+### Within the SIMP Omni-Environment
+
+The `simp_bolt` module is compatible with the SIMP Omni-Evironment to apply
+SIMP settings to target systems.  To utilize a SIMP Omni-Environment, copy the
+contents of `/usr/share/simp/environment-skeleton/puppet` to the desire Puppet
+environment directory, which will also serve as the Boltdir. Copy the contents
+of `/usr/share/simp/environment-skeleton/secondary` and 
+`/usr/share/simp/environment-skeleton/writable` to the desired destinations.
+The paths for all three of the enviroments, as well as the environment name,
+are configurable in Hiera. After copying the environment skeletons, change
```

Comment:
How does the configurable writable directory work?  The simplib::passgen code which writes to the writable directory looks up the Puppet[:vardir] setting and builds the writable directory from there.  So, that directory has to be in a specific place.

Reply:
The cache dir  where the writable writable skeleton goes defaults to /opt/puppetlabs/puppet/cache for root and ~/.puppetlabs/opt/puppet/cache for non-root users, so we should be able to place the writable environment there and "it should just work".

Reply:
So, does it make sense to have the writable path fully configurable in the manifest?  In reality on the cache dir for your set up works.  I would not want the user to think they could put the writable directory anywhere.